### DISCUSSION TITLE
20240110-java-and-kotlin/

### DISCUSSION THREAD

**Discussion by giscus** [inflearn/tech-blog-comment#47] [Comments]
# 20240110-java-and-kotlin/

Spring Boot & JPA 환경에서 Java와 Kotlin을 함께 사용한 이유와 사용 예제를 소개합니다.

https://tech.inflab.com/20240110-java-and-kotlin/

**Comment by mystyle2006** [comment]
아직 코틀린/스프링 쪽을 경험해보지 못했는데 이런 내용들이 있군요..

**Comment by ACACIANh** [comment]
많은 실험 및 정보 감사합니다, 은총알은 없네요ㅠㅠ
